build/verify-features.ts

```diff
@@ -20,15 +20,15 @@ for (let fileName of featuresDirContents) {
 		continue;
 	}
 
-	if (!importedFeatures.includes(featureMeta.id)) {
-		errors.push(`ERR: ${fileName} should be imported by \`/sources/refined-github.ts\`.`);
+	const featureId = fileName.replace('.tsx', '');
+	if (!importedFeatures.includes(featureId)) {
+		errors.push(`ERR: ${featureId} should be imported by \`/sources/refined-github.ts\`.`);
```

Comment:
No period at the end, which would not have been there had you clicked the "accept suggestion" instead 😃

Reply:
Ahh yes sorry about that